docs/source/contribute/migration/0_x-to-1_0.rst

```diff
@@ -0,0 +1,28 @@
+Migrating from 0.x to 1.0
+=========================
+
```

Comment:
Please add information on how to know what the theme version you're running is. 
Either by looking at the footer or by looking at the pyproject.toml.